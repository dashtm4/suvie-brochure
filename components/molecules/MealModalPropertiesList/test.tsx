import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { MealModalPropertiesList } from '.'

describe('<MealModalPropertiesList />', () => {
  const children = 'Test children'

  it('should renders correct <MealModalPropertiesList />', () => {
    const wrapper = shallow(
      <MealModalPropertiesList>{children}</MealModalPropertiesList>
    )

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
