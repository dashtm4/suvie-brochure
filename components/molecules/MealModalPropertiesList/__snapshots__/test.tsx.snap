// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MealModalPropertiesList /> should renders correct <MealModalPropertiesList /> 1`] = `
<ul
  className=""
>
  Test children
</ul>
`;
